lib/team_statistics.rb

```diff
@@ -0,0 +1,267 @@
+class TeamStatistics
+  PERIODS = ['last_day', 'last_week', 'last_month', 'last_year']
```

Comment:
I've implemented the frontend period select but noticed the the ticket specifies these time windows:

- Past Week
- Past 2 Weeks
- Past Month (this is  the default set)
- Past 3 Months
- Past 6 Months
- Year to Date

Reply:
@amoedoamorim , ranges updated:

• past_week
• past_2_weeks
• past_month
• past_3_months
• past_6_months
• year_to_date